packages/frontend-2/components/workspace/ProjectList.vue

```diff
@@ -32,9 +32,30 @@
             v-bind="bind"
             v-on="on"
           />
-          <FormButton v-if="!isWorkspaceGuest" @click="openNewProject = true">
-            New project
-          </FormButton>
+          <div class="flex gap-2">
```

Comment:
@Mikehrn It seems that v-tippy is not reactive. When you go from a workspace page that you are not an admin of, to one you are an admin, the buttons will correctly be enabled, but the tooltip remains.

Conditionally rendering the surrounding div seems to fix this, but open to suggestions if you have any other ideas.